src/components/Cards/restaurantCard.tsx

```diff
@@ -1,23 +1,69 @@
 import * as React from "react";
-import "./restaurantCard.css";
+import styled from "styled-components";
+// import "./restaurantCard.css";
+
+const RestarauntContainer = styled.section`
+  width: 360px;
+  overflow: hidden;
+  margin: 1vh;
```

Comment:
This can yield unexpected results
**Great article on using viewport units** - https://www.sitepoint.com/css-viewport-units-quick-start/

Reply:
Thanks!